Symfony/CS/Fixer/PSR2/BracesFixer.php

```diff
@@ -605,4 +606,27 @@ private function getFinalControlContinuationTokensForOpeningToken($openingTokenK
 
         return array();
     }
+
+    private function fixDeclareStrictTypes(Tokens $tokens)
+    {
+        $declares = $tokens->findGivenKind(T_DECLARE);
+
+        foreach ($declares as $index => $declare) {
+            if ($this->isStrictTypeDeclare($index, $tokens)) {
```

Comment:
Hi @keradus, sorry, on a short holiday atm. From what I remember, just the declare strict types block is different and should be formatted like `declare(strict_types=1);`. Other declares could be used like other block elements and should be rendered like this,

```php
declare (ticks = 200) {
    //... 
} 
```

Perhaps any `declare` on a single line should be formatted without spacing? e. g. `declare(ticks=200);` in which case the check should really be for single/block declares.